doc/changes/changes-4.10.xml

```diff
@@ -0,0 +1,312 @@
+<Chapter Label="ChangesGAP49toGAP410"> 
+<Heading>Changes between &GAP; 4.9 and &GAP; 4.10</Heading>
+
+This chapter contains an overview of the most important changes
+introduced in &GAP; 4.10.0 release (the 1st public release of &GAP; 4.10).
+Later it will also contain information about subsequent update 
+releases for &GAP; 4.10.
+<P/>
+
+These changes are also listed on the Wiki page
+<URL>https://github.com/gap-system/GAP/wiki/gap-4.10-release-notes</URL>.
+
+<Section Label="GAP4.10.1">
+<Heading>&GAP; 4.10.0 (November 2018)</Heading>
+
+<Subsection> 
+<Heading>New features and major changes</Heading>
+
+<List>
+
+<!-- 
```

Comment:
why such empty comments here? (and below)

Reply:
Helped me to navigate the document.